src/utils.c

```diff
@@ -2188,3 +2188,91 @@ void utils_start_new_geany_instance(const gchar *doc_path)
 	else
 		g_printerr("Unable to find 'geany'");
 }
+
+
+static GFile *utils_gfile_create(const gchar *fname)
+{
+	if (utils_is_uri(fname))
+		return g_file_new_for_uri(fname);
+	return g_file_new_for_path(fname);
+}
+
+
+/**
+ *  Tests file existence using the configured IO method in Geany.
+ *
+ *  @param fname File name.
```

Comment:
Do we actually  want those in the API?  It's cute, but do plugin really care about that?